assets/js/navigation.js

```diff
@@ -3,13 +3,13 @@
 ( function( $ ) {
 
 	var $navMenu    = false,
-	    $menuToggle = false,
-	    $submenu    = false;
+	    $menuToggle = false;
 
 	function toggle() {
 
 		$menuToggle.add( $navMenu ).toggleClass( 'open' );
 
+		// @todo Can this be removed? It seems somewhat annoying that the expanded submenus would reset when toggling the menu.
 		resetSubmenuToggles();
```

Comment:
That's a good point. I can see how one would expect the submenus to remain expanded when toggling the menus. Let's get rid of this.

Reply:
Removed in 4d36f81